content/sensu-go/5.15/installation/platforms.md

```diff
@@ -103,7 +103,7 @@ Docker images containing the Sensu backend and Sensu agent are available for Lin
 ## Building from source
 
 Sensu Go's core is open source software, freely available under an MIT license.
-Sensu Go instances built from source do not include some free-tier and licensed-tier features such as the web UI homepage; see the [feature comparison matrix][15] to learn more.
+Sensu Go instances built from source do not include some free and commercial features such as the web UI homepage; see the [feature comparison matrix][15] to learn more.
```

Comment:
It might not be necessary to make a distinction between free and commercial (previously licensed-tier) features, because the commercial features are offered for free.

Reply:
Thank you -- got it!